readthedocs/projects/tasks.py

```diff
@@ -1310,6 +1310,20 @@ def warn(self, msg):
     delete_queryset.delete()
 
 
+@app.task(max_retries=5, default_retry_delay=7 * 60)
+def clean_build_task(version_id):
```

Comment:
`clean_build_artifacts` is a better name, I think.

(no need to use the `_task` suffix as well)

Reply:
`_task` suffix is used in all the code. There are more things than tasks in `projects/tasks`, that's why.

Reply:
build artifacts makes reference to the build output, not virtual env/conda/checkout

Reply:
The docsting says that it cleans the build artifacts ;)

Reply:
There 2 tasks ending with `_task` and 14 (also celery tasks) without it in that file. I'd follow the majority in this case :)